package.json

```diff
@@ -114,18 +114,18 @@
     "babel-loader": "^8.1.0",
     "babel-plugin-dynamic-import-node": "^2.3.3",
     "babel-plugin-styled-components": "1.11.1",
-    "browserslist": "^4.14.0",
+    "browserslist": "^4.14.5",
     "chalk": "^4.1.0",
     "enquirer": "^2.3.6",
-    "eslint": "^7.3.1",
+    "eslint": "^7.10.0",
     "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^6.12.0",
     "eslint-plugin-es5": "^1.5.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-json": "^2.1.2",
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "^7.20.6",
+    "eslint-plugin-react": "7.20.6",
```

Comment:
The removal of the caret here is temporary to prevent a version bump to 7.21.2 which contains breaking lint changes.

This issue looks like it has been fixed in 7.22.2 (not yet released) https://github.com/yannickcr/eslint-plugin-react/issues/2804